File: C#/Урок 036/Задача 12.cs
class Program
{
    static void Main()
    {
        System.Console.Write("Enter initial sum: ");
        decimal initialSum = System.Convert.ToDecimal(System.Console.ReadLine());

        System.Console.Write("Enter years: ");
        int years = System.Convert.ToInt32(System.Console.ReadLine());

        System.Console.Write("Enter deposit: ");
        decimal deposit = System.Convert.ToDecimal(System.Console.ReadLine());

        decimal sum = initialSum;
        for (int year = 1; year <= years; year++)
            sum += deposit;

        System.Console.WriteLine($"You will get {sum:C} in {years} years");
    }
}

File: C++/Урок 032 - while/Задача 12.cpp
#include <iostream>

int main()
{
    std::cout << "Enter deposit: $";
    double deposit;
    std::cin >> deposit;

    std::cout << "Enter amount of years: ";
    int years;
    std::cin >> years;

    const double INITIAL_DEPOSIT = deposit;
    const double INCREASE_FACTOR = 0.05;

    int year = 1;
    while (year <= years)
    {
        deposit += deposit + deposit * INCREASE_FACTOR;
        std::cout << "You will get $" << deposit << " in " << year << " years.\n";
        year++;
    }

    return 0;
}